rocAL/rocAL_pybind/amd/rocal/plugin/pytorch.py

```diff
@@ -193,15 +170,15 @@ def __next__(self):
 
         else:
             if(self.loader._oneHotEncoding == True):
-                self.loader.GetOneHotEncodedLabels(self.labels, self.device)
-                self.labels_tensor = self.labels.view(-1, self.bs, self.loader._numOfClasses).long()
+                print("Support for OneHotEncodedLabels is not added")
```

Comment:
Revert this change

Reply:
Reverted. But this API will not work as expected